{{cookiecutter.repo_name}}/docs/conf.py

```diff
@@ -260,7 +269,7 @@
 # html_search_scorer = 'scorer.js'
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = '{{ cookiecutter.repo_name }}doc'
+htmlhelp_basename = '{project_name}doc'.format(project_name='{{ cookiecutter.project_name }}')
```

Comment:
Nit: Can this line be:

```python
htmlhelp_basename = '{{ cookiecutter.project_name }}doc'
```

Or if it needs to be dynamically updatable, define `project_name` above under [line 88](https://github.com/edx/cookiecutter-django-app/pull/9/files#diff-a7c00ab2c7691e433aa60ba11a09d277R88):

```python
project_name = '{{ cookiecutter.project_name }}'
```

 and use it like you've used the others:

```python
htmlhelp_basename = '{project_name}doc'.format(project_name=project_name)
```